src/scss/components/pages.scss

```diff
@@ -279,11 +279,37 @@ body {
     }
 
     &.active {
-      color: $white;
+      color: $gray-500;
 
       &::before {
         color: $white;
       }
     }
   }
 }
+
+.listing__item--search-result {
+  .breadcrumb {
+
+    > .breadcrumb-item {
+
+      a {
+        color: $gray-500;
```

Comment:
I think this violates accessibility contrast requirements (50% grey on white?)

Reply:
Have changed to grey-800